src/core/annotation.js

```diff
@@ -1610,11 +1610,14 @@ class ChoiceWidgetAnnotation extends WidgetAnnotation {
       }
     }
 
-    // Determine the field value. In this case, it may be a string or an
-    // array of strings. For convenience in the display layer, convert the
-    // string to an array of one string as well.
-    if (!Array.isArray(this.data.fieldValue)) {
+    // The field value can be `null` if no item is selected, a string if one
+    // item is selected or an array of strings if multiple items are selected.
+    // For consistency in the API and convenience in the display layer, we
+    // always make the field value an array with zero, one or multiple items.
+    if (isString(this.data.fieldValue)) {
       this.data.fieldValue = [this.data.fieldValue];
```

Comment:
If this is a String, shouldn't you also wrap the value in `stringToPDFString` to prevent issues elsewhere?
```suggestion
    if (isString(this.data.fieldValue)) {
      this.data.fieldValue = [stringToPDFString(this.data.fieldValue)];
```